packages/medusa/src/api-v2/store/orders/[id]/route.ts

```diff
@@ -0,0 +1,13 @@
+import { MedusaRequest, MedusaResponse } from "../../../../types/routing"
+import { refetchOrder } from "../helpers"
+
+// TODO: Do we want to apply some sort of authentication here? My suggestion is that we do
```

Comment:
We haven't really had much previously, but it might make sense to add. So far all you really need is the order ID.